.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -39,6 +39,7 @@ body:
       label: What Exporter(s) are you seeing the problem on?
       multiple: true
       options:
+        - stdout
```

Comment:
nit - In Memory too, even thought it's for internal use, bugs can still be raised.